pygments/lexers/arturo.py

```diff
@@ -0,0 +1,312 @@
+import re
+
+from pygments.lexer import bygroups, default, DelegatingLexer, do_insertions,\
+    include, RegexLexer, this, using, words
+from pygments.token import Comment, Generic, Keyword, Name, Number, Operator,\
+    Other, Punctuation, String, Text
+
+from pygments.util import ClassNotFound, get_bool_opt
+
+class ArturoLexer(RegexLexer):
```

Comment:
please add a docstring with `versionadded`, look at other modules for examples

Reply:
- [x] [add ArturoLexer's docstring](https://github.com/pygments/pygments/pull/2259/commits/fc625b63f15ce90e329c3523438a0cd63dec0473)